core/src/scene/scene.h

```diff
@@ -74,6 +76,8 @@ class Scene {
     auto& background() { return m_background; }
     auto& fontContext() { return m_fontContext; }
     auto& globals() { return m_globals; }
+    auto& globalUpdates() { return m_globalUpdates; }
```

Comment:
Seems like this is really a collection of global _references_, I think we should call it something like that now that we've figured out exactly how this works.